resources/views/hardware/bulk-checkin.blade.php

```diff
@@ -0,0 +1,115 @@
+@extends('layouts/default')
+
+ Page title
+@section('title')
+  {{ trans('admin/hardware/general.checkin') }}
+  @parent
+@stop
+
+ Page content
+@section('content')
+  <style>
+
+    .input-group {
+      padding-left: 0px !important;
+    }
+  </style>
+
+
+  <div class="row">
+    <!-- left column -->
+    <div class="col-md-9">
+      <div class="box box-default">
+        <div class="box-header with-border">
+          <h2 class="box-title">{{trans('admin/hardware/form.bulk_checkin')}}</h2>
```

Comment:
This translation string got removed I think (it shows up `asset-bulk-actions.blade.php` as well

![CleanShot 2023-09-20 at 11 40 38@2x](https://github.com/snipe/snipe-it/assets/1141514/9aa985ba-324b-40dc-9899-30541901c6eb)